inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -163,6 +163,13 @@ def pdf_validator(form, field):
     if field.data and not is_pdf_link(field.data):
         raise StopValidation(message)
 
```

Comment:
As [`Travis complains`](https://travis-ci.org/inspirehep/inspire-next/jobs/262670672#L1786), we enforce the fact that there are two newlines between functions, like PEP8 (https://www.python.org/dev/peps/pep-0008/#blank-lines) says:
> Surround top-level function and class definitions with two blank lines.